main.py

```
@@ -1,20 +1,310 @@
\ No newline at end of file
+def save(path, data):
+    with open(path, "w", encoding="utf-8") as f:
+        json.dump(data, f, indent=2, ensure_ascii=False)
+
+def is_admin(uid):
+    return uid in load(ADMINS_DB)
+
+async def is_member(bot, uid):
+    try:
+        m = await bot.get_chat_member(FORCE_CHANNEL, uid)
+        return m.status in ["member", "administrator", "creator"]
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)